packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -375,14 +372,19 @@ export class Tabs extends GOVUKFrontendComponent {
    * @param {KeyboardEvent} event - Keydown event
    */
   onTabKeydown(event) {
-    switch (event.keyCode) {
-      case this.keys.left:
-      case this.keys.up:
+    switch (event.key) {
+      // 'Left', 'Right', 'Up' and 'Down' required for Edge 16 support.
```

Comment:
If we wanted to save a few bytes, we could ignore this and just go with the values in the spec. It shouldn't cause errors – it'd just mean that navigating between the tabs using the keyboard wouldn't work in Edge 16.

The number of users visiting GOV.UK (based on those who've opted in to analytics) are _vanishingly_ small – between 20 to 50 sessions per day in February (out of 95,335,104 – so about 0.00004% of our sessions).

Thoughts?

Reply:
We discussed this in our dev catch up and decided that because the only way to move between tabs using the keyboard is using the arrow keys we should keep this in to properly support Edge 16.